tests/lib.rs

```diff
@@ -732,3 +732,40 @@ fn retain() {
     vs.sort();
     assert_eq!(v, &*vs);
 }
+
+#[test]
+fn get_one_short() {
+    let x = ('x', 42);
+
+    let (r, mut w) = evmap::new();
+
+    w.insert(x.0, x);
+
+    assert_match!(r.get_one(&x.0), None);
+
+    w.refresh();
+
+    assert_match!(r.get_one(&x.0).unwrap().as_ref(), ('x', 42));
+}
+
+#[test]
+fn get_one_long() {
+    let x = ('x', 42);
```

Comment:
I kept this value this way for consistency but didn't want to just add this same value every time. Wanted to sort of document the fact that the output value is not determined. Am open to:
1. inserting `x` every time
1. changing/removing `x` because we don't need the tuple
1. anything else!

Reply:
I would probably change this to `let key = 'x';` :)

Reply:
Much more readable! Changes in f804389bf60096f8d563173a6c143ad4ce30557b